spec/features/work_packages/share_spec.rb

```diff
@@ -134,15 +129,15 @@
       click_button 'Share'
 
       # These users were not changed
-      share_modal.expect_shared_with(view_user, 'View')
-      share_modal.expect_shared_with(comment_user, 'Comment')
-      share_modal.expect_shared_with(shared_project_user, 'Edit')
-      share_modal.expect_shared_with(current_user, editable: false)
+      share_modal.expect_shared_with(comment_user, 'Comment', position: 1)
       # This user's role was updated
-      share_modal.expect_shared_with(not_shared_yet_with_user, 'Comment')
+      share_modal.expect_shared_with(not_shared_yet_with_user, 'Comment', position: 2)
+      share_modal.expect_shared_with(shared_project_user, 'Edit', position: 3)
+      share_modal.expect_shared_with(current_user, position: 4, editable: false)
+      share_modal.expect_shared_with(view_user, 'View', position: 5)
```

Comment:
👍 for those changes to the specs. Quite nice to have that behaviour safeguarded.